servers/services/pom.xml

```diff
@@ -81,10 +99,71 @@
       <artifactId>junit-jupiter-api</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-params</artifactId>
+      <scope>test</scope>
+    </dependency>
     <dependency>
       <groupId>org.assertj</groupId>
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
Looks like a lot of dependencies that are needed to test Nessie w/ JAX-RS.
I think, it would be easier to have a separate pom.xml that pulls in these dependencies so the modules that wanna test against in-JVM Nessie/Jersey/Weld just pull in a single dependency.